src/components/Todo/common/Todo/CompletedList/index.tsx

```diff
@@ -46,3 +67,37 @@ export const TodoCompletedItem: FC<Props> = ({
 
   return null;
 };
+
+
+export interface TodoCompletedListProps {
+  items: TodoItem[];
+  completedItems: TodoItem[];
+  setItemsCallback: (updatedItems: TodoItem[]) => void;
+}
+
+export const TodoCompletedList: FC<TodoCompletedListProps> = ({
+  completedItems,
+  items,
+  setItemsCallback
+}) => {
+  const completedItemsLength = completedItems.length;
+
+  if(completedItemsLength === 0) return null;
+
+  return (
+    <>
```

Comment:
No need of this fragment here. Please remove

Reply:
Have updated please check